### PR TITLE
[13.0][MIG] calendar

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration of `calendar`.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---Models in module 'calendar'---
+---Fields in module 'calendar'---
+calendar     / calendar.alarm           / alarm_type (selection)        : NEW required, selection_keys: ['email', 'notification'], req_default: function, hasdefault
+calendar     / calendar.alarm           / type (selection)              : DEL required, selection_keys: ['email', 'notification'], req_default: function
+# DONE: pre-migration: renamed fields
+
+calendar     / calendar.event           / event_tz (selection)          : NEW selection_keys: function, hasdefault
+# NOTHING TO DO: New field that has default, will be automatically filled
```
Comment (kos94ok-3D) [inline addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt:8]: I don`t know why but I have a null value in an event_tz column after migration.
Do you have the same problem?

Comment (MiquelRForgeFlow) [reply addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt:8]: I will check. But null value in this field means that `self.env.context.get('tz') or self.user_id.tz` is falsy 🤔

Comment (MiquelRForgeFlow) [reply addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt:8]: Well, the field is not required, so no serious problem if it has null values.

Comment (kos94ok-3D) [reply addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt:8]: You are right:
self.env.context.get('tz') == None
and
self.user_id == res.users() => self.user_id.tz == False

**Review by kos94ok-3D — APPROVED** [review]

File: addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (pedrobaeza) [reply addons/calendar/migrations/13.0.1.0/openupgrade_analysis_work.txt:8]: I'm pushing a revision making sure this field is empty for all for preserving v12 behavior.

**Review by pedrobaeza — APPROVED** [review]
With my added changes

**Pull request merged**